gprofiler/profilers/java.py

```diff
@@ -683,7 +685,15 @@ def _run_fdtransfer(self) -> None:
             # run fdtransfer with accept timeout that's slightly greater than the jattach timeout - to make
             # sure that fdtransfer is still around for the full duration of jattach, in case the application
             # takes a while to accept & handle the connection.
-            [fdtransfer_path(), self._fdtransfer_path, str(self.process.pid), str(self._jattach_timeout + 5)],
+            [
+                asprof_path(),
+                "fdtransfer",
+                "--fd-path",
+                self._fdtransfer_path,
+                "--timeout",
+                str(self._jattach_timeout + 5),
+                str(self.process.pid),
```

Comment:
For `fdtransfer` and `jattach` verbs, you're appending `pid` last to the arguments list. For `jcmd` it's first. Any reason? Let's remain consistent

Reply:
Right. async-profiler is perhaps permissive, and accepts pid anywhere.